arkouda/strings.py

```diff
@@ -89,7 +91,8 @@ def __init__(self, offset_attrib : Union[pdarray,np.ndarray],
             self.ndim = self.offsets.ndim
             self.shape = self.offsets.shape
         except Exception as e:
-            raise ValueError(e)
+            raise ValueError(e)   
+        self.dtype = self.bytes.dtype
```

Comment:
doesn't this need to change to `self.dtype = np.str_` or something

Reply:
LOL, yeah, git fail on the part of your favorite contractor.